docsets/AWS_CloudFormation_Template_Reference/docset.json

```diff
@@ -1,10 +1,16 @@
 {
     "name": "AWS CloudFormation Template Reference",
-    "version": "2023.03.20",
+    "version": "20241230.01",
     "archive": "AWS_CloudFormation_Template_Reference.tgz",
```

Comment:
This file doesn't exist. I think you added `AWS_CloudFormation_Template_Reference.tgz.txt` instead. Please note that txt file is just a file left over by my processing script. You should add the archive as `AWS_CloudFormation_Template_Reference.tgz`.

Reply:
Just amended the commit to fix that.